Hippo.Web/Controllers/AdminController.cs

```diff
@@ -127,6 +127,96 @@ public async Task<IActionResult> RemoveClusterAdmin(int id)
         return Ok();
     }
 
+    #region Financial Admin methods
+    [HttpGet]
+    public async Task<IActionResult> FinancialAdmins()
+    {
+        // get all users with cluster admin permissions
+        return Ok(await _dbContext.Users
+            .AsNoTracking()
+            .Where(u => u.Permissions.Any(p => p.Cluster.Name == Cluster && p.Role.Name == Role.Codes.FinancialAdmin))
+            .OrderBy(u => u.LastName).ThenBy(u => u.FirstName)
+            .ToArrayAsync());
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> AddFinancialAdmin(string id)
```

Comment:
seems very repetitive w/ add/remove cluster admin, almost copy/paste could probably be combined

Reply:
Yeah... I got lazy, it is a little different query...